web/src/features/datasets/server/dataset-router.ts

```diff
@@ -120,6 +121,16 @@ export const datasetRouter = createTRPCRouter({
             JOIN observations o ON o.id = ri.observation_id
           WHERE o.project_id = ${input.projectId}
           group by 1
+        ),
+        avg_total_cost_by_run_id AS (
```

Comment:
This could be merged with latency_by_run_id, observation_view includes all columns of observations